apparmor.d/abstractions/lxqt

```diff
@@ -0,0 +1,31 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 Besanon <m231009ts@mailfence.com>
+# SPDX-License-Identifier: GPL-2.0-only
+
+  abi <abi/4.0>,
+
+  include <abstractions/fonts>
+  include <abstractions/freedesktop.org>
+  include <abstractions/gtk>
+  include <abstractions/qt5>
+  include <abstractions/wayland>
+  include <abstractions/X-strict>
+
+  signal (receive) set=(kill, term) peer=lxqt-session,	
+
+  /usr/share/hwdata/pnp.ids 	  r,
+  /usr/share/icu/@{int}.@{int}/*.dat 			r,
+  /usr/share/lxqt/**    				r,
+  /usr/share/qt{5,6}/ 					r,
+  /usr/share/qt{5,6}/{,**} 			r,
+ 
+  owner @{HOME}/.Xdefaults 			r,
+
+  owner @{user_cache_dirs}/fontconfig/* 		      rw,
+  owner @{user_cache_dirs}/lxqt-notificationd/*  	r,
+
+  owner @{user_config_dirs}/lxqt/*.conf			      rw,
+
```

Comment:
I am not sure if allowing full config access to all program using this abs (ie: a lot) is the best option. I think we can start this way and restrict this a bit later.